### DISCUSSION TITLE
What is the whole logic behind simplify_below ?

### DISCUSSION THREAD

**Discussion by BenOnTrack** [systemed/tilemaker#709] [Q&A]
Hi, 

I'm trying to optimize the size of my .mbtiles, knowing that 40% ends up in the landcover layer (with the openmaptiles config).
I've been playing around with simplify_below/level/ratio and I'm not quite sure to understand how that works, despite reading the CONFIGURATION.md 

Any tip on how to properly set those parameters (for polygons, such as tiles ending up in landcover) will be appreciated.

**Comment by systemed** [comment]
Generally:

- set `simplify_below` to the first zoom level where you want unsimplified geometries (often 13 or 14)
- set `simplify_level` to a small fraction of a degree of longitude, e.g. 0.0003
- don't worry too much about `simplify_ratio` (you can just use the default) or `simplify_length` (which is just another way of expressing simplify_level)

As you zoom out, you will probably also want to filter out small polygons/parts of polygons entirely. Do this with something like

    "filter_below": 12, "filter_area": 0.5

Experiment with the `simplify_level` and `filter_area` settings until you get a result you're happy with.

Finally, consider setting `combine_polygons_below` on any landuse/landcover layers to dissolve adjoining features together.

**Comment by BenOnTrack** [comment]
Hi Richard, thanks for the heads up.  So I've been playing around quite a lot with those parameters (based on the .json config examples provided under resources) and did not notice notable changes. I've mostly been able to define a threshold below which features are actually so simplified that they completely disappear. 

As far as I remember, MapLibre/Box comes with a max tiles size for performance during rendering. I used to be able to run https://github.com/ibesora/vt-optimizer to analyze quickly which Layer were wrong - I just can't make it work anymore (probably due to my poor background with node.js and my recent node upgrade :( and I though that could potentially help me identifying where my tilemaker process needs to be improved
